---
description: LOCALDB_ERROR_INTERNAL_ERROR
title: LOCALDB_ERROR_INTERNAL_ERROR | Документация Майкрософт
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 60c7a47b-114c-441b-869e-9d8ebfe4148e
author: stevestein
ms.author: sstein
ms.openlocfilehash: 018e5b33e3da26a3412a671666e6b95a4817a1d2
ms.sourcegitcommit: e700497f962e4c2274df16d9e651059b42ff1a10
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 08/17/2020
ms.locfileid: "88491010"
---
# <a name="localdb_error_internal_error"></a>LOCALDB_ERROR_INTERNAL_ERROR
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>Сведения  
  
| attribute | Значение |
| --------- | ----- |
|Название продукта|SQL Server|  
|Идентификатор события|264|  
|Источник события|Среда выполнения локальной базы данных SQL Server 12.0|  
|Компонент|API среды выполнения локальной базы данных|  
|Текст сообщения|Произошла непредвиденная ошибка внутри вызова метода API экземпляра локальной базы данных. Сведения об ошибке см. в журнале приложений Windows.|  
  
## <a name="explanation"></a>Объяснение  
 Произошла непредвиденная ошибка.  
  
## <a name="user-action"></a>Действие пользователя  
 Подробные сведения см. в журнале событий.  
  
  
